diffkemp/simpll/MacroUtils.h

```diff
@@ -39,14 +39,14 @@ struct MacroElement {
     std::string sourceFile;
 };
 
-/// Difference between macro in left module and the same macro in the right
-/// module.
-struct MacroDifference {
-	// Name of the macro.
-	std::string macroName;
+/// Syntactic difference between objects that cannot be found in the original
+/// source files
```

Comment:
I'd add here a note that these differences can be either in macros or in inline assemblies.